server/domain/academic-record/models/AcademicRecord.ts

```
@@ -0,0 +1,182 @@
+    studentId: string | null;
+    semesters: {
+      year: number;
+      semester: number;
+      attemptedCredits: number | null;
+      earnedCredits: number | null;
+      semesterGpa: number | null;
+      semesterPercentile: number | null;
+      attemptedCreditsGpa: number | null;
+      classRank: number | null;
+      totalStudents: number | null;
+    }[];
```

ESLint

[error] Array type using 'T[]' is forbidden for non-simple types. Use 'Array<T>' instead.

(@typescript-eslint/array-type)